.sphinx/_static/custom.css

```diff
@@ -13,8 +13,14 @@ div.page, h1, h2, h3, h4, h5, h6, .sidebar-tree .current-page>.reference, button
 th.head {
     text-transform: uppercase;
     font-size: var(--font-size--small);
+    text-align: initial;
 }
 
+table.align-center th.head {
+    text-align: center
+}
+
+
```

Comment:
Nitpick/Question: Are these two blank lines intended?

Reply:
Nope, I just didn't pay attention.
Do you want to update it or should I?

Reply:
Okay, I changed it :)